Assets/FluidBehaviorTree/Tests/Editor/Decorators/RepeatForeverTest.cs

```diff
@@ -0,0 +1,40 @@
+using CleverCrow.Fluid.BTs.Decorators;
+using CleverCrow.Fluid.BTs.Tasks;
+using NUnit.Framework;
+
+namespace CleverCrow.Fluid.BTs.Testing {
+    public class RepeatForeverTest {
+        public class UpdateMethod {
```

Comment:
Move repeated logic into a `[Setup]` method. Mostly just the `repeater` declaration and creation.